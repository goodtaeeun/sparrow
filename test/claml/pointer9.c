int zip(int a, int b) { return 0; }
int unzip(int a, int b) { return 0; }

int (*work)(int a, int b) = zip;

int main() {
  work = unzip;
  sparrow_print(*work);
  return 0;
}
