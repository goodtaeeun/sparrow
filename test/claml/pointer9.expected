/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 1 "pointer9.c"
int zip(int a, int b) {

  {
#line 1
    return (0);
  }
}
#line 2 "pointer9.c"
int unzip(int a, int b) {

  {
#line 2
    return (0);
  }
}
#line 4 "pointer9.c"
int (*work)(int, int) = &zip;
#line 6 "pointer9.c"
int main(void) {

  {
    {
#line 7
      work = &unzip;
#line 8
      sparrow_print(&*work);
    }
#line 9
    return (0);
  }
}
